README.md

```diff
@@ -2,6 +2,17 @@
 
 A Base64 encoding/decoding library written in Noir which can encode arbitrary byte arrays into Base64 and decode Base64-encoded byte arrays (e.g. `"SGVsbG8gV29ybGQ=".as_bytes()`).
 
+## Noir Version Compatibility
+
+We have tested this library with Noir nightly, 0.36.0, and 1.0.0-beta.0
+
+## Dependencies
+
+- Noir >=1.0.0
+- Barretenberg ≥v0.56.1
+
+Refer to [Noir's docs](https://noir-lang.org/docs/getting_started/quick_start) and [Barretenberg's docs](https://github.com/AztecProtocol/aztec-packages/blob/master/barretenberg/cpp/src/barretenberg/bb/readme.md#installation) for installation steps.
```

Comment:
```suggestion
## Noir Version Compatibility

We have tested this library with Noir nightly, 0.36.0, and 1.0.0-beta.0

Refer to [Noir's docs](https://noir-lang.org/docs/getting_started/quick_start) for installation steps.
```

As a general rule, I don't think that any noir libararies should be specifying versions of the backend to use (or even mentioning barretenberg at all).

The reasoning being that the version of the backend which the user should use is purely dependent on the version of nargo they're using (and unaffected by the library) so it should be documented elsewhere. If we add references to barretenberg here then we need to keep this up-to-date across N different libraries which is creating more work for ourselves.